### PR TITLE
Add query template redirect link to query

### PR DESCRIPTION
**Pull request by yiwenwang-ms**
Add query template redirect link to query

### CODE REVIEW

File: api-reference/beta/api/search-query.md [hunk 1]
```diff
@@ -160,13 +163,126 @@ Content-type: application/json
 }
 ```
 
+### Example 2: Basic call to use queryTemplate.
+
+The following is an example of the request shows how to use a queryable property CreateBy to retrieve all files which created by Bob.
```
Comment (Lauragra) [inline api-reference/beta/api/search-query.md:168]: Revise as:
The following example shows how to use the queryable property **createBy** to retrieve all files created by a user.

File: api-reference/beta/api/search-query.md [hunk 2]
```diff
@@ -160,13 +163,126 @@ Content-type: application/json
...
+        "queryTemplate":"{searchTerms} CreatedBy:Bob"
+      },
+      "from": 0,
+      "size": 25
+    }
+  ]
+}
+```
+
+# [C#](#tab/csharp)
```
Comment (Lauragra) [inline api-reference/beta/api/search-query.md:199]: Please remove all the snippet tabs from the new example, including the one on line 172. These will get automatically generated via the weekly snippet generation.

File: api-reference/beta/api/search-query.md [hunk 3]
```diff
@@ -55,14 +55,17 @@ If successful, this method returns an `HTTP 200 OK` response code and a collecti
 
 ## Examples
 
-### Request
+### Example 1: Basic call to perform a search request.
+
+The following is an example of the request shows how to perform a search to get expected connector items.
```
Comment (Lauragra) [inline api-reference/beta/api/search-query.md:60]: Revise as:
The following example shows how to perform a search to get expected connector items.

Comment (yiwenwang-ms) [reply api-reference/beta/api/search-query.md]: Thank you! Done

File: api-reference/beta/api/search-query.md (hunk 2 above)
Comment (yiwenwang-ms) [reply api-reference/beta/api/search-query.md]: Thank you! Done

File: api-reference/beta/api/search-query.md [hunk 4]
```diff
@@ -160,13 +163,126 @@ Content-type: application/json
...
+
+# [PHP](#tab/php)
+[!INCLUDE [sample-code](../includes/snippets/php/search-query-php-snippets.md)]
+[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
+
+---
+
+### Response
+
+The following is an example of the response.
```
[the remark this answers is not in the document]
Comment (yiwenwang-ms) [reply api-reference/beta/api/search-query.md]: Thank you! Done

**Review by Lauragra — APPROVED** [review]

**Pull request merged**